queries.sql

```diff
@@ -10,3 +10,37 @@ SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE NOT name='Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+/* p2 */
+/* ROLEBACK UPDATE */
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+/* COMMIT */
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
```

Comment:
- [ ] please add a query to check that the `species` col was changed before the commit and a query to check that it is committed after the `COMMIT`

Reply:
Thanks for your tips